src/helpers/types.ts

```diff
@@ -175,7 +177,14 @@ export type Payload = {
   delay: number;
   gracePeriod: number;
   payloadsController: Hex;
-  actionAddresses: Hex[];
+  actions: readonly {
```

Comment:
better to just use the abi inferred type? I guess this way could be unsured it's aligned